### PR TITLE
BZ-1752832: Mention to add to noProxy if workers are scaled up

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@lamek @bobfuru Can one of you peer review this please?

**Comment by bergerhoffer** [comment]
@wzheng1 For review please.

@danehans Can you also review and make sure this is okay?

Preview: 
* https://bz-1752832--ocpdocs.netlify.com/openshift-enterprise/latest/networking/enable-cluster-wide-proxy.html#nw-proxy-configure-object_config-cluster-wide-proxy
* https://bz-1752832--ocpdocs.netlify.com/openshift-enterprise/latest/machine_management/manually-scaling-machineset.html#machineset-manually-scaling_manually-scaling-machineset

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1752832

File: modules/machineset-manually-scaling.adoc [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 If you must add or remove an instance of a machine in a MachineSet, you can manually scale the MachineSet.
 
+[NOTE]
+====
+If you have enabled the cluster-wide proxy and scale up workers after installation, you must add the workers to the Proxy object's `noProxy` field to prevent connection issues.
```
Comment (kalexand-rh) [inline modules/machineset-manually-scaling.adoc]: ~have~

Do you have an assembly about modifying the proxy? If so, I suggest adding a link to it and a reminder about this need to the next steps for the scale up assemblies.

Comment (bergerhoffer) [reply modules/machineset-manually-scaling.adoc]: @kalexand-rh I would love to add a link, but this is in a module so I can't here.

Comment (bergerhoffer) [reply modules/machineset-manually-scaling.adoc]: Sorry @kalexand-rh, I was so cranky about no xrefs in modules that I didn't fully read your suggestion about link being in the next steps of the assembly. I'll look at this.

Comment (kalexand-rh) [reply modules/machineset-manually-scaling.adoc]: I understand the xref crankiness, but I think that the end of the assembly's the best place for that link anyway. :)

**Comment by vikram-redhat** [comment]
Ack received in BZ. Merging.

**Pull request merged**